tool/tsh/db.go

```diff
@@ -188,46 +183,92 @@ func onDatabaseLogout(cf *CLIConf) {
 }
 
 func databaseLogout(tc *client.TeleportClient, db tlsca.RouteToDatabase) error {
-	// First perform database-specific actions, such as remove connection
-	// information from Postgres service file.
-	switch db.Protocol {
-	case defaults.ProtocolPostgres:
-		err := pgservicefile.Delete(tc.SiteName, db.ServiceName)
-		if err != nil {
-			return trace.Wrap(err)
-		}
+	// First remove respective connection profile.
+	err := dbprofile.Delete(tc, db)
+	if err != nil {
+		return trace.Wrap(err)
 	}
 	// Then remove the certificate from the keystore.
-	err := tc.LogoutDatabase(db.ServiceName)
+	err = tc.LogoutDatabase(db.ServiceName)
 	if err != nil {
 		return trace.Wrap(err)
 	}
 	return nil
 }
 
-// onDatabaseEnv handles "tsh db env" command.
+// onDatabaseEnv implements "tsh db env" command.
 func onDatabaseEnv(cf *CLIConf) {
+	tc, err := makeClient(cf, false)
+	if err != nil {
+		utils.FatalError(err)
+	}
+	database, err := pickActiveDatabase(cf)
+	if err != nil {
+		utils.FatalError(err)
+	}
+	env, err := dbprofile.Env(tc, *database)
+	if err != nil {
+		utils.FatalError(err)
+	}
+	for k, v := range env {
+		fmt.Printf("export %v=%v\n", k, v)
+	}
```

Comment:
You may not need this, but if you want to print some instructions as well, you can add something like the following:

```go
fmt.Printf(`echo 'Environment loaded, run the following command to connect...\n'`)
```

I'm doing something similar for `tsh login leaf.example.com --env`.

Reply:
The purpose of this command is for its output to be fed to shell's eval (like, `$(tsh db env)` - similar to what minikube for example does as well). For easier discovery of the connect commands I added those to `tsh db ls` table.